site/pt/blog/new-in-devtools-111/index.md

```diff
@@ -113,39 +135,50 @@ Chromium issues: [1407586](https://crbug.com/1407586), [1402891](https://crbug.c
 {# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/0a7c428980b8f8a60e83c780e17f6a6f94007493 #} 
 {# https://chrome-internal.googlesource.com/devtools/devtools-internal/+/aea51aad7026881ad26ca1b270d12338adb92d79 #} 
 
-Chromium issue: [1339771](https://crbug.com/bbb)
+Issue relacionada: [1339771](https://crbug.com/bbb)
 
 
 <!-- ## Better syntax highlight for Angular {: #syntax } -->
+## Melhor syntax highlight para Angular {: #syntax }
 
 <!-- DevTools enhanced the syntax highlighting for Angular HTML templates, making it easier for you to read code and recognize its structure. -->
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/lJU7wOa4vNDb2Vm2zPJq.png", alt="Syntax highlighting for the Angular HTML templates.", width="800", height="507" %}
+O DevTools aprimorou o realce de sintaxe para templates HTML do Angular, facilitando a leitura do código e o reconhecimento de sua estrutura.
```

Comment:
"realce de sintaxe" / "syntax highlight" - we seem to sometimes use the translated term and others the english one. I'd recommend keeping it consistent.

Reply:
```suggestion
O DevTools aprimorou o sintaxe highlighting para templates HTML do Angular, facilitando a leitura do código e o reconhecimento de sua estrutura.
```